### PR TITLE
Add fe filter for jinja2 templates

### PR DESCRIPTION
**Pull request by bobsilverberg**
As per @glogiotatidis's suggestion at https://github.com/mozilla/oneanddone/pull/301#discussion_r43484696

r?

### CODE REVIEW

**Comment by cargoworker** [comment]
Can one of the admins verify this patch?

**Comment by glogiotatidis** [comment]
ok to test

**Comment by glogiotatidis** [comment]
@bobsilverberg thanks for the pull request! Can you please make use of the filter here https://github.com/mozilla/sugardough/blob/master/%7B%7B%20cookiecutter.project_name%20%7D%7D/%7B%7B%20cookiecutter.project_name%20%7D%7D/base/templates/%7B%7B%20cookiecutter.project_name%20%7D%7D/home.jinja#L43 and make sure that the template is rendered correctly here https://github.com/mozilla/sugardough/blob/master/%7B%7B%20cookiecutter.project_name%20%7D%7D/%7B%7B%20cookiecutter.project_name%20%7D%7D/base/tests.py#L9?

**Comment by bobsilverberg** [comment]
Thanks for the review @glogiotatidis. I have added a test in my latest commit.

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/tests.py [hunk 1]
```diff
@@ -8,6 +9,10 @@ def test_base(self):
         response = self.client.get(reverse('home'))
         assert b'csrfmiddlewaretoken' in response.content
 
+    def test_fe_filter(self):
+        response = self.client.get(reverse('home'))
+        assert 'DEBUG setting: {0}'.format(settings.DEBUG) in response.content
```
Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/tests.py]: I think you need a `b` like in https://github.com/mozilla/sugardough/pull/98/files#diff-3fed87ec34a6808ba36f7d63ed41b42cR10 to make this work in python3

**Comment by glogiotatidis** [comment]
Tests are failing for python3

**Comment by bobsilverberg** [comment]
@glogiotatidis I believe all the tests are passing now. :)

**Comment by bobsilverberg** [comment]
@glogiotatidis: @robhudson commented [1] in the One and Done commit where I added this that there is, iho, a better solution [2] than this. Would you mind taking a look at that, and if you agree that we should not add the `fe` filter to sugardough feel free to close this PR.

[1] https://github.com/mozilla/oneanddone/pull/301/files#r45286842
[2] https://bugzilla.mozilla.org/show_bug.cgi?id=1222040

**Comment by glogiotatidis** [comment]
I'm super happy that somehow Will's comment reached oneanddone, now sugardough and in a moment lumbergh's https://github.com/mozilla/lumbergh/pull/140. A perfect example of how much open development rocks. You're all awesome! 

Yes, newstyle gettext works better so I'm closing the PR. Thanks @bobsilverberg :gem:

**Pull request closed without merging**